docs/INSTALL.MD

```diff
@@ -28,34 +100,54 @@ JuliaBox is developed and tested on Ubuntu 14.04. But it should be easy to adapt
         ]
     }
     </pre>
-    - Run `JuliaBox/scripts/install/jbox_configure.sh` to generate JuliaBox configuration files.
-    - Run `sudo JuliaBox/scripts/install/mount_fs.sh /jboxengine/data 3 500 ${USER}` to create loopback devices for JuliaBox user home volumes.
-5. Pull/build docker images:
-    - Pull/build base docker images, by following instructions at:
-        - Pull the base ubuntu image (only if building). Run: `docker pull ubuntu:14.04`
-        - <https://github.com/tanmaykm/JuliaDockerImages/tree/master/base/v0.3>
-        - <https://github.com/tanmaykm/JuliaDockerImages/tree/master/pkgdists/juliaboxminimal>
-        - <https://github.com/tanmaykm/JuliaDockerImages/tree/master/pkgdists/juliabox>
-    - Build JuliaBox images:
-        - Julia and default package bundles. Run: `JuliaBox/scripts/install/img_create.sh build /jboxengine/data`
-        - JuliaBox components: `JuliaBox/scripts/install/img_create.sh jbox`
-6. Create database. Run: `JuliaBox/scripts/install/create_tables_sqlite.py /jboxengine/data/db/juliabox.db`
-7. Run JuliaBox:
+    - Generate JuliaBox configuration files:
+		- Run `JuliaBox/scripts/install/jbox_configure.sh`.
+	- Create disk images, loopback devices, and mount:
+		- Run `sudo JuliaBox/scripts/install/mount_fs.sh /jboxengine/data 3 500 ${USER}` 
+
+3. JuliaBox *containers* Docker image creation.
+	
+	- Acquire a base Julia image, containing the packages you wish to be pre-installed. This can be achieved by pulling and existing image, or building from a specified Dockerfile.
+
+		- To pull an image:
+			- [Review available images](https://github.com/tanmaykm/JuliaDockerImages)
+			- Pull selected image, e.g., `docker pull julialang/juliabox:v0.3.11`
+			
+		- To build an image:
+			- Run `docker pull ubuntu:14.04`.
+			- Clone template Dockerfiles: `git clone https://github.com/tanmaykm/JuliaDockerImages.git`.
+			- Follow instructions in repository to build and tag images.
+	
+	- Build JuliaBox images:
+		- Modify `FROM` in `JuliaBox/docker/Dockerfile` to point to the correct base image.
```

Comment:
Should also probably mention how to change the bundled default packages:

- If the chosen image is not the default JuliaBox image (`julialang/juliaboxpkgdist`), you may also remove undesired packages from `JuliaBox/docker/setup_julia.sh` or add new ones of your choice.